broker-cli/utils/grpc-deadline.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Grpc Deadline
+ * @module broker-daemon/utils/grpc-deadline
+ */
+
+/**
+ * @constant
+ * @type {number}
+ * @default
+ */
+const DEFAULT_TIMEOUT_IN_SECONDS = 5
+
+/**
+ * gRPC uses the term `deadline` which is a timeout feature that is an absolute
+ * point in time, instead of a duration.
+ *
+ * @param {number} [timeoutInSeconds=DEFAULT_TIMEOUT_IN_SECONDS]
+ * @returns {number} deadline in seconds
```

Comment:
I think this should return `Date`

Reply:
@bradleystachurski the type is actually a number

```
> typeof new Date().setSeconds(new Date().getSeconds() + 5)
'number'
```

Reply:
Ahh, I misinterpreted the the docs on this method that it is `an absolute point in time` to refer to a `Date` object. The docs for grpc specify `deadline` (https://grpc.io/grpc/node/grpc.html#~Deadline) can be either `number | Date`, so disregard my comment.

Reply:
@bradleystachurski No worries. I should have elaborated on the comment as well. 

Just to be thorough, we return a number here, but signify its use in the comment. We use this same pattern in a few places where we return a Date string (ISO8601) and say `@returns {string} date string in ISO8601 format`) 

However... if I was actually returning a Date object (`typeof new Date` === `object`) then I would expect that we use `@returns {Date}` since the return value has methods that are related to the Date object for JS